gui/api/resources.py

```diff
@@ -352,6 +352,10 @@ def dehydrate(self, bundle):
             bundle.data['stg_guicertificate'] = bundle.obj.stg_guicertificate.id
         else:
             bundle.data['stg_guicertificate'] = None
+
+        bundle.data['stg_guiaddress'] = bundle.obj.stg_guiaddress.split()
+        bundle.data['stg_guiv6address'] = bundle.obj.stg_guiv6address.split()
```

Comment:
Can you also massage the API to be backward compatible, accepting values as string?